### PR TITLE
Release 2.2.0-beta3

### PR DESCRIPTION
**Pull request by niik**
## Overview

This is the release PR for the third beta release of the 2.2.0 series.

## Description

The main addition here is onboarding tutorial improvements and bug fixes.

## TODO

- [x] Still need to merge #8360 
- [x] Still need to merge #8374

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "2.2.0-beta3": [
+      "[Fixed] Fix file path truncation in merge conflicts dialog - #6666",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] File path truncation in merge conflicts dialog - #6666",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+      "[Fixed] Support pushing workflow files to GitHub.com - #7079",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Support pushing workflow files for GitHub Actions to GitHub.com - #7079",
```

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+      "[Fixed] Resume tutorial if new repo has been added - #8341",
+      "[Fixed] Retain commit summary placeholder in tutorial repo - #8354",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Retain commit summary placeholder in tutorial repo - #8354",
```

**Comment by outofambit** [comment]
we done shipped this

**Pull request merged**